.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "realm/realm-library/src/main/cpp/object-store"]
+	path = realm/realm-library/src/main/cpp/object-store
+	url = git@github.com:realm/realm-object-store.git
```

Comment:
FWIW, we've preferred to use the HTTPS URLs in our other projects as not all users have SSH keys configured on GitHub.